src/CarbonAware.WebApi/src/Models/SciScoreInput.cs

```diff
@@ -6,19 +6,51 @@ namespace CarbonAware.WebApi.Models;
 [Serializable]
 public record SciScoreInput
 {
+    /// <example> 
+    /// {
+    ///     "locationType": "CloudProvider",
+    ///     "latitude": 0,
```

Comment:
We should remove lat/long from these examples.  If we want to show lat/long we need a 2nd example (I don't know if swagger supports that)